raddb/radiusd.conf.in

```diff
@@ -56,7 +56,7 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 sysconfdir = @sysconfdir@
-localstatedir = @localstatedir@
+localstatedir = "" # this places the PID in /run which is the correct path in systemd.
```

Comment:
This change is completely wrong.  This file is an *input* file which is used by `configure` to create the actual `radiusd.conf`.  The `localstatedir`, etc. values are taken from the `configure` arguments.  So if you want to change the values, change the arguments to `configure`.  There's no need to break the `radiusd.conf.in` file for everyone.

Reply:
The debian/freeradius.service file IS NOT intended to be used when you build the server yourself from source.  It's intended to be used with debian packages.

Reply:
My gut feeling told me this was wrong, and instead of bashing you could pint me in the right direction. But no you didn't.

Reply:
@enoch85 Arran's comments weren't "bashing" you.  They were factual and straightforward.  I suggest apologizing for insulting someone who gave you factual advice.

Reply:
@enoch85 I was emulating the RFC requirements level style, i.e. SHOULD, SHOULD NOT, MUST, MUST NOT, SHALL, SHALL NOT etc...  See RFC 2119.